js/main.js

```diff
@@ -14,11 +14,135 @@
   var MAX_COORDINATE_Y = 630;
   var PRICE = 10000;
 
-  // Скрываем лэйаут карты
+  // Вводим переменные
   var map = document.querySelector('.map');
-  map.classList.remove('map--faded');
   var mapPinsArea = document.querySelector('.map__pins');
```

Comment:
```suggestion
  var mapPinsArea = map.querySelector('.map__pins');
```
https://up.htmlacademy.ru/javascript/20/criteries#d21

> Поиск элементов по селекторам делается минимальное количество раз, после этого ссылки на элементы сохраняются.